src/commoncode/datautils.py

```diff
@@ -36,17 +36,6 @@
 Utilities and helpers for data classes.
 """
 
-# Python 2 and 3 support
```

Comment:
This commit is repetition of https://github.com/nexB/scancode-toolkit/pull/1534/commits/1df2b4947cee0919cb2bda8c6bd009f327d68b2e .